docs/Navigation/SetVideoConfig/assets/SetVideoConfigBasicFlow.puml

```diff
@@ -9,8 +9,8 @@ Participant "SDL" as SDL
 Participant "HMI" as HMI
 
 A -> SDL: StartService(serviceType:Video, streamingParams)
-SDL -> HMI: UI.SetVideoConfig(config: streamingParams)
-HMI -> SDL: UI.SetVideoConfig(SUCCESS)
+SDL -> HMI: Navigation.SetVideoConfig(config: streamingParams)
+HMI -> SDL: Navigation.SetVideoConfig(SUCCESS)
 SDL -> A: StartServiceACK(streamingParams)
```

Comment:
Since this is a protocol message and not an RPC it should be probably be presented a little differently
```suggestion
SDL -->> A: StartServiceACK(streamingParams)
```